hs_explore/migrations/0002_auto_20200625_1910.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-06-25 19:10
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def fill_csdms_names(apps, schema_editor):
+    """ Store CSDMS names into LDAWord model
+    """
+    LDAWord = apps.get_model('hs_explore', 'LDAWord')
+    CSDMSName = apps.get_model('hs_csdms', 'CSDMSName')
+    csdms_names = set()
+    splitted_names = set()
+    for csdms_record in CSDMSName.objects.filter(part='name'):
+        csdms_name = csdms_record.value
+        csdms_names.add(csdms_name)
+        if len(csdms_name) <= 1:
+            continue
+        lda_word_record = LDAWord()
+        lda_word_record.source = 'CSDMS'
+        lda_word_record.word_type = 'keep'
+        lda_word_record.part = 'name'
+        lda_word_record.value = csdms_name
+        lda_word_record.save()
+        tokens = csdms_name.split(" ")
+        splitted_names.update(tokens)
+
+    for splitted_name in splitted_names:
+        if len(splitted_name) <= 1 or splitted_name in csdms_names:
+            continue
+        lda_word_record = LDAWord()
+        lda_word_record.source = 'CSDMS'
+        lda_word_record.word_type = 'keep'
+        lda_word_record.part = 'name'
+        lda_word_record.value = splitted_name
+        lda_word_record.save()
+
+
+def fill_odm2(apps, schema_editor):
+    """ Store ODM2 names into LDAWord model
+    """
+    ODM2Variable = apps.get_model('hs_odm2', 'ODM2Variable')
+    LDAWord = apps.get_model('hs_explore', 'LDAWord')
+    term_names = ODM2Variable.objects.all().values_list('name').order_by('name')
+    odm2_list = [str(t[0].replace(",", " -")) for t in term_names if not t[0][0].isdigit()]
+    modified_list = []
+    for odm2 in odm2_list:
+        tokens = odm2.split(' - ')
+        new_string = tokens[0]
+        if len(tokens) > 1:
+            new_string = tokens[1] + ' ' + tokens[0]
+        modified_list.append(new_string.lower())
+
+    for odm2_name in modified_list:
+        lda_word_record = LDAWord()
+        lda_word_record.source = 'ODM2'
+        lda_word_record.word_type = 'keep'
+        lda_word_record.part = 'name'
+        lda_word_record.value = odm2_name
+        lda_word_record.save()
+
+
+def fill_stop_words(apps, schema_editor):
+    """ Store common English and customized stop-words into the LDAWord model
+    """
+    LDAWord = apps.get_model('hs_explore', 'LDAWord')
+    customized_stops = ['max', 'maximum', 'minimum', 'origin', '________', 'center',
```

Comment:
hardcoding these lists in a migration seems shortsighted.  I'd imagine we'd want some flexibility in tweaking the stop words

Reply:
Yes. I agreed. Besides hardcoding these lists, the only way I could think about is to pre-store them in some files as I have done for the csdms standard names list. Is this a better way to deal with these two lists?